### PR TITLE
Updated widgets docs for 5.1

### PR DESCRIPTION
**Pull request by kubama**

### CODE REVIEW

File: content/working_with/console/widgets/deployments.md [hunk 1]
```diff
@@ -8,8 +8,8 @@ Displays the list of the deployments in the current tenant, according to the use
 
 ![blueprint-deployments]( /images/ui/widgets/blueprint-deployments.png )
 
-Each deployment in the list includes details about the attached blueprint, when the deployment was created (if it was updated icon indicating that will be shown near creation date),
-which site it is assigned to and the nodes on which it is deployed.
+Each deployment in the list includes details about the attached blueprint, when the deployment was created (if it was updated icon indicating that will be shown near creation date)
```
Comment (qooban) [inline content/working_with/console/widgets/deployments.md:11]: If I remember correctly Site Name is presented, but only when it is set.

File: content/working_with/console/widgets/executions.md [hunk 2]
```diff
@@ -42,3 +42,4 @@ In the actions menu on the right side of the execution row (click ![List icon](
 You can also choose to add `Id` column from the list, which will present the execution id. By default this value is not presented as a column in the table, but as a pop up shown by hovering over ID label.
 
 * `Show system executions`- allow to include or exclude executions of system workflows in the list. Default: On
+* `Show most recent execution only` - if enabled the widget only shows a tasks graph for the most recent execution
```
Comment (qooban) [inline content/working_with/console/widgets/executions.md:45]: Maybe we can improve Executions widget documentation to provide some details about how to read and use tasks graph? There's nothing about tasks graph in the UI documentation as for now, I think. We could add a screenshot at least and description how to interpret it.

File: content/working_with/console/widgets/plugins.md [hunk 3]
```diff
@@ -8,6 +8,7 @@ Displays a list of all the plugins uploaded to the current tenant, according to
 
 The widget displays the following information:
 
+* **Plugin name**
```
Comment (qooban) [inline content/working_with/console/widgets/plugins.md:11]: Possibly we can also add plugin icon here.

**Review by qooban — COMMENTED** [review]
Added few comments.

File: content/working_with/console/widgets/deployments.md (hunk 1 above)
Comment (kubama) [reply content/working_with/console/widgets/deployments.md]: Right - reverted.

File: content/working_with/console/widgets/plugins.md (hunk 3 above)
Comment (kubama) [reply content/working_with/console/widgets/plugins.md:12]: Done

File: content/working_with/console/widgets/executions.md (hunk 2 above)
Comment (kubama) [reply content/working_with/console/widgets/executions.md:56]: Done

**Review by ofercloudify — APPROVED** [review]

**Pull request merged**